docs/reference/management_commands.rst

```diff
@@ -125,3 +125,12 @@ search_garbage_collect
     $ ./manage.py search_garbage_collect
 
 Wagtail keeps a log of search queries that are popular on your website. On high traffic websites, this log may get big and you may want to clean out old search queries. This command cleans out all search query logs that are more than one week old (or a number of days configurable through the :ref:`WAGTAILSEARCH_HITS_MAX_AGE <wagtailsearch_hits_max_age>` setting).
+
+rm_cropped_images
```

Comment:
I suggest this is renamed to `remove_image_renditions`, or `remove_generated_images`